sdks/python/apache_beam/transforms/stats_test.py

```diff
@@ -216,11 +221,13 @@ def test_approximate_unique_global_by_sample_size_with_small_population(self):
 
   @unittest.skipIf(sys.version_info < (3, 0, 0),
                    'Skip with py27 because hash function is not good enough.')
+  @retry(reraise=True, stop=stop_after_attempt(3))
```

Comment:
I saw that we retry only in 2 test cases - is that intentional or we need to retry other test cases as well?

Reply:
Other tests are pretty stable and I used fixed input, so it would be OK. However, given randomness of hash function, is it better to add retries to all tests?

Reply:
If particular values of hash() can cause these test to fail, we should retry them as well, but if not, we don't have to.